src/main/resources/services/upcomingReleases/upcomingReleases.ts

```diff
@@ -18,10 +18,12 @@ exports.get = (req: Request): Response => {
   const statistics: Array<StatisticInListing> = getAllStatisticsFromRepo()
   const upComingReleases: Array<Release> = getUpcomingReleases(statistics)
   const count: number = req.params.count ? parseInt(req.params.count) : 2
+  const showAll: boolean = req.params.showAll && req.params.showAll === 'true' ? true : false
```

Comment:
It should be enough to write this off as const showAll: boolean = !!req.params.showAll.